Logger/SerilogConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuickEdit.LoggerConfig;
```

Comment:
Use the same namespace as `AutoLog.cs`, as there isn't really a reason to separate the two
```suggestion
namespace QuickEdit.Logger;
```